### PR TITLE
Add dictionary definition for 'Aesthetic'

### PR DESCRIPTION
**Pull request by shagunZ**
## This PR fixes...
The lack of dictionary definition and perspective for 'Aesthetic'

## What I did...

-Add dictionary definition for aesthetic
-Add dictionary perspective for aesthetic

## How to test...

_In a list format, tell us how to test this PR._
Ex:

1. Navigate to /dictionary
1. Search "Aesthetic"
1. See definition and perspective show up under term "aesthetic"
2. check grammar and types

## I learned...

As a beginner I learned how to contribute to open source repository.

### CODE REVIEW

**Comment by shagunZ** [comment]
can someone please tell me what got wrong ? why it is'nt acceptable?

File: src/assets/content/dictionary/aesthetic.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ---
 title: Aesthetic
-definition:
-sources: []
-perspectives: []
+definition:"Aesthic is a particular conception of beauty or art, a particular taste for or approach to what is pleasing to the senses and especially sight"
```
Comment (novellac) [inline src/assets/content/dictionary/aesthetic.md:3]:
```suggestion
definition: Aesthic is a particular conception of beauty or art, a particular taste for or approach to what is pleasing to the senses and especially sight.
```
Bug: I think the lack of a space between `definition:` and `Aesthetic` is breaking the build. You can accept this suggestion via the Github UI, or you can make an additional commit to this PR, as you like.

**Review by novellac — CHANGES_REQUESTED** [review]
Almost there! I think the suggestion I made will unbreak the build. I'll be happy to approve and merge when we get the build working.

**Comment by novellac** [comment]
Closing, as I've just merged in your previous PR for the same term, #179

**Pull request closed without merging**